### PR TITLE
hotfix/remove-unused-kernels-from-jupyterlab

### PR DESCRIPTION
**Pull request by mthiboust**
# Description

Remove unused kernels from jupyterlab

### CODE REVIEW

File: docker/jupyterlab/Dockerfile [hunk 1]
```diff
@@ -41,6 +41,10 @@ RUN umask 002 && \
     jupyter lab build --dev-build=False && \
     jupyter lab clean 
 
+# Remove unused kernels
+# ---------------------
+RUN jupyter kernelspec remove xpython xpython-raw -y
```
Comment (pieeri) [inline docker/jupyterlab/Dockerfile:46]: ❓ 

Could we remove them directly by scrapping the `xeus-python` dep in the conda/mamba config?

Comment (mthiboust) [reply docker/jupyterlab/Dockerfile:46]: What do you mean by "scrapping"? Doing a `mamba remove xeus xeus-python xeus-python-shell xeus-python-shell-raw xeus-zmq`? It isn't the documented way to do this, but I can try. FYI, those packages are very light (2 Mb in total, so we won't gain substancial space with this)

```
xeus                      5.1.1                h4795217_0    conda-forge
xeus-python               0.17.1          py311hcddfd71_0    conda-forge
xeus-python-shell         0.6.3              pyhd8ed1ab_0    conda-forge
xeus-python-shell-raw     0.6.3              pyhd8ed1ab_0    conda-forge
xeus-zmq                  3.1.0                h0da9c92_0    conda-forge
```

Comment (mthiboust) [reply docker/jupyterlab/Dockerfile]: I confirm that `mamba remove xeus` also remove the XPython kernels from `jupyter kernelspec list` and from the `/opt/conda/share/jupyter/kernels/` folder. New commit uses this method instead.

Comment (mthiboust) [reply docker/jupyterlab/Dockerfile]: Ok, my bad, I haven't seen that this package was explicitly mentionned in the `environment.yml` file. Simply removing it from there now.

**Review by pieeri — APPROVED** [review]
👍

**Pull request merged**